CONTRIBUTING.md

```diff
@@ -29,3 +29,78 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+The general steps for adding features to the format are as follows:
+
+1. Discuss changes on on the developer mailing list (dev@parquet.apache.org).  Often times it is helpful to link to a draft pull request to make the discussion concrete. This step is complete when there lazy consensus.
+
+2. One a change has lazy consensus two implementations of the feature
+demonstrating interopability must also be provided.  One implementation MUST be [parquet-java](http://github.com/apache/parquet-java).  It is preferred that the second implementation be [parquet-cpp](https://github.com/apache/arrow) or [parquet-rs](https://github.com/apache/arrow-rs), however at the discretion of the PMC any
```

Comment:
Does this mean features are implemented on main branch or released or we want to stay ambiguous and see what works in practice?

Reply:
IMHO, it should be on the main branch. It is not common practice to port new (unstable) features to a maintenance branch.

Reply:
Agree, I was assuming main branch.

Reply:
tried to clarify.  PTAL.